images/bridge/cmd/bridge/main.go

```diff
@@ -367,6 +374,14 @@ func (m *SessionHandler) serveSessionText(w http.ResponseWriter, r *http.Request
 }
 
 func (m *SessionHandler) serveSessionUpdates(w http.ResponseWriter, r *http.Request) {
+	if !websocket.IsWebSocketUpgrade(r) {
+		w.Header().Set("Content-Type", "application/json")
+		json.NewEncoder(w).Encode(map[string]any{
+			"session_start":    m.Session.Start,
+			"event_stream_url": m.EventStreamURL,
+		})
```

Comment:
This is used for now to generate some initial values used in the client.
The session start time should probably be generated as an event when a new session is created. It would probably make sense to use the "conflict keys" or whatever the current naming is to just send that at startup.

But this also needed some way to configure the stream url to include the "space" and bridge id, which at the moment seemed easier to do server-side as well.